config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
```

Comment:
Since we don't use actioncable we might be able to get away with leaving this out, but probably fine as is. Let's just make sure it isn't raising any errors when we do normal stuff.